CODEOWNERS

```diff
@@ -110,6 +110,7 @@ drivers/usb/                             @jfischer-phytec-iot @finikorg
 drivers/usb/device/usb_dc_stm32.c        @ydamigos @loicpoulain
 drivers/i2c/i2c_ll_stm32*                @ldts @ydamigos
 dts/arm/st/                              @erwango
+dts/arm/nordic                           @ioannisg @carlescufi
```

Comment:
I think you want a trailing slash here

Reply:
right, thanx.
Fixed (let's wait for @carlescufi to have a look)